dataminer-overview/FAQ/FAQs_Support_Services.md

```diff
@@ -22,9 +22,11 @@ uid: FAQs_Support_Services
   
 - **What happens when my renewal of my support plan is overdue (Perpetual-Use plan only)?**
 
-  By default, support plan contracts renew automatically every year, unless they have been stopped by the user in advance of the anniversary date of the agreement. For more details, see [Pricing - Term](xref:Support_services_terms#pricing---term).
+  By default, support plans renew automatically every year, unless they have been stopped by the user in advance of the end date of the plan.
 
-  In case the support plan has not been deactivated, and the user failed to renew the program in time by sending a valid PO, on the day after the anniversary date of the agreement, the service (i.e. Continuity or Evolve plan) will automatically be stopped along with all related services linked to it. As a user, you can still continue to consult content and resources on the Dojo Community portal.
+  Skyline will provide you with a renewal proposal for the support services at least three months prior to the end date of your support plan. Unless you provide Skyline with a written notice of your wish not to renew the support plan no later than thirty days prior to the end date of the plan, it is automatically renewed for another year, and Skyline will invoice the amount of the renewal proposal. All amounts due to Skyline are payable up front before the provision of any support services. If you terminate the support services early, this does not entitle you to a refund.
+
+  In case the support plan has not been deactivated, and you fail to timely issue pay your invoice(s) or a PO (if required), on the day after the end date of the plan, the service may be stopped along with all related services linked to it. However, you can still continue to consult content and resources on the Dojo Community portal.
```

Comment:
Something seems off about this sentence. Can I rephrase it like this?

```suggestion
  In case the support plan has not been deactivated, and you fail to pay your invoice(s) or issue a PO (if required) in time, on the day after the end date of the plan, the service may be stopped along with all related services linked to it. However, you can still continue to consult content and resources on the Dojo Community portal.
```

Reply:
perfect Marieke, thank you